modin/pandas/series.py

```diff
@@ -1948,15 +1948,22 @@ def to_numpy(
         """
         Return the NumPy ndarray representing the values in this Series or Index.
         """
-        return (
-            super(Series, self)
-            .to_numpy(
-                dtype=dtype,
-                copy=copy,
-                na_value=na_value,
+        from modin.config import ExperimentalNumPyAPI
+
+        if not ExperimentalNumPyAPI.get():
+            return (
+                super(Series, self)
+                .to_numpy(
```

Comment:
What about the changes for `.values`?

Reply:
`values` calls `self.to_numpy()` and then does some post processing, so I think that should be covered by `to_numpy` returning a modin array!